receiver/nginxreceiver/scraper.go

```diff
@@ -73,12 +73,13 @@ func (r *nginxScraper) scrape(ctx context.Context) (pdata.ResourceMetricsSlice,
 	}
 
 	metrics.AddSumDataPoint(metadata.M.NginxRequests.Name(), stats.Requests)
-	metrics.AddGaugeDataPoint(metadata.M.NginxConnectionsActive.Name(), stats.Connections.Active)
 	metrics.AddSumDataPoint(metadata.M.NginxConnectionsAccepted.Name(), stats.Connections.Accepted)
 	metrics.AddSumDataPoint(metadata.M.NginxConnectionsHandled.Name(), stats.Connections.Handled)
-	metrics.AddGaugeDataPoint(metadata.M.NginxConnectionsReading.Name(), stats.Connections.Reading)
-	metrics.AddGaugeDataPoint(metadata.M.NginxConnectionsWriting.Name(), stats.Connections.Writing)
-	metrics.AddGaugeDataPoint(metadata.M.NginxConnectionsWaiting.Name(), stats.Connections.Waiting)
+
+	metrics.WithLabels(map[string]string{metadata.L.State: "active"}).AddGaugeDataPoint(metadata.M.NginxConnectionsCurrent.Name(), stats.Connections.Active)
```

Comment:
active, reading, writing, etc. will be generated once they're enumed in metadata.yaml.

Reply:
@jrcamp thanks for the review, I wasn't aware of that option in the metadata. I've updated the metadata as you directed!